Src/BootCamp.Chapter/Equipment.cs

```diff
@@ -7,85 +7,60 @@ namespace BootCamp.Chapter
     public class Equipment
     {
-        private Weapon _weapon;
-        public void SetWeapon(Weapon weapon)
-        {
-
-        }
-
-        private Headpiece _head;
-        public void SetHead(Headpiece head)
-        {
-
-        }
-
-        private Chestpiece _chest;
-        public void SetChest(Chestpiece chestpiece)
-        {
-
-        }
-
-        private Shoulderpiece _leftShoulder;
-        public void SetLeftShoulder(Shoulderpiece should)
-        {
-
-        }
-
-        private Shoulderpiece _rightShoulder;
-        public void SetRightShoulder(Shoulderpiece shoulder)
-        {
+        public Weapon Weapon { get; set; }
+        public Headpiece Head { get; set; }
+        public Chestpiece Chest { get; set; }
+        public Shoulderpiece LeftShoulder { get; set; }
+        public Shoulderpiece RightShoulder { get; set; }
+        public Legspiece Legs { get; set; }
+        public Armpiece LeftArm { get; set; }
+        public Armpiece RightArm { get; set; }
+        public Gloves Gloves { get; set; }
 
-        }
-
-        private Legspiece _legs;
-        public void SetLeg(Legspiece legs)
-        {
-
-        }
-
-        private Armpiece _leftArm;
-        public void SetLeftArmp(Armpiece arm)
+        public float GetTotalWeight()
         {
+            var items = new Item[] { Weapon, Head, Chest, LeftShoulder, RightShoulder, Legs, LeftArm, RightArm, Gloves };
+            float totalWeight = 0;
 
-        }
+            foreach (var item in items)
+            {
+                if (item == null)
+                {
+                    continue;
+                }
 
-        private Armpiece _rightArm;
-        public void SetRightArm(Armpiece arm)
-        {
+                totalWeight += item.Weight;
+            }
 
+            return totalWeight;
         }
 
-        private Gloves _gloves;
-        public void SetGloves(Gloves gloves)
+        public float GetTotalDefense()
         {
+            var items = new Armor[] { Head, Chest, LeftShoulder, RightShoulder, Legs, LeftArm, RightArm, Gloves };
+            float totalDefence = 0;
 
-        }
+            foreach (var item in items)
+            {
+                if (item == null)
+                {
+                    continue;
+                }
 
-        /// <summary>
-        /// Gets total weight of armour.
-        /// </summary>
-        /// <returns></returns>
-        public float GetTotalWeight()
-        {
-            return 0;
-        }
+                totalDefence += item.Defence;
+            }
 
-        /// <summary>
-        /// Returns sums of defense stat of all armour pieces.
-        /// </summary>
-        /// <returns></returns>
-        public float GetTotalDefense()
-        {
-            return 0;
+            return totalDefence;
         }
 
-        /// <summary>
-        /// Returns damage done by weapon.
-        /// </summary>
-        /// <returns></returns>
         public float GetTotalAttack()
```

Comment:
Try to forget about get methods, unless they have rich logic. 
```cs
public float TotalAttack
{
  get
  {
                if (Weapon == null)
            {
                return 0;
            }

            return Weapon.Attack;
  }
}
```
is totally fine